apps/aeutils/priv/aeternity_config_schema.json

```diff
@@ -1762,6 +1762,11 @@
                                                     "description": "The number of blocks in a parent generation.",
                                                     "type": "integer"
                                                 },
+                                                "retry_interval": {
```

Comment:
Seems very similar to `polling > retry_interval` below?!

Reply:
Thanks, I'll move it into the polling section.